README.md

```diff
@@ -94,7 +94,7 @@ You can help to translate the bot using [Hosted Weblate](https://hosted.weblate.
 
 ## Using the public bot
 
-The [public bot](https://t.me/RSStT_Bot) comes with absolutely no warranty. I will try my best to maintain it, but I cannot guarantee that it will always work perfectly. Meanwhile, you should "fair use" the bot, avoid subscribing to too many RSS feeds.  
+The [public bot](https://t.me/RSStT_Bot) comes with absolutely no warranty. I will try my best to maintain it, but I cannot guarantee that it will always work perfectly. Meanwhile, you should "fair use" the bot, avoid subscribing to too many RSS feeds.
```

Comment:
Oops, these twin trailing whitespaces are used to add linebreaks. I will amend your last commit to replace them with `\`.